azext_iot/central/_help.py

```diff
@@ -212,6 +212,45 @@ def _load_central_users_help():
             --object-id {objectId}
             --role operator
     """
+    helps[
+        "iot central app user show"
+    ] = """
+    type: command
+    short-summary: Get the details of a user by ID
+    examples:
+      - name: Get details of user
+        text: >
+          az iot central app user show
+          --app-id {appid}
+          --id {userId}
```

Comment:
FYI - I'm not sure if this ID is actually surfaced anywhere in our UI. Customers may have a hard time using this if their entire flow is not CLI based

Reply:
CLI get list will get you the list of ID's